decidim-participatory_processes/app/helpers/decidim/participatory_processes/participatory_process_helper.rb

```diff
@@ -21,6 +21,20 @@ def step_dates(participatory_process_step)
         dates = [participatory_process_step.start_date, participatory_process_step.end_date]
         dates.map { |date| date ? localize(date.to_date, format: :default) : "?" }.join(" - ")
       end
+
+      def assemblies_for_participatory_process(participatory_process_assemblies)
+        html = ""
+        html += %( <div class="section"> ).html_safe
```

Comment:
It could be possible use a card instead of having hardcoded html in this helper please?